cspell.json

```diff
@@ -52,6 +52,12 @@
         "uninstantiated",
         "urem",
         "vecmap",
+        "direnv",
+        "nixpkgs",
+        "envrc",
+        "phated",
+        "mybranch",
```

Comment:
"mybranch"  being here indicates that a variable should be named "my_branch" or "myBranch" somewhere

Reply:
Found the usage in readme :)